darwin/importer/importer.py

```diff
@@ -0,0 +1,106 @@
+from pathlib import Path
+from typing import Callable, List, Optional, Union
+
+import darwin.datatypes as dt
+from darwin.utils import secure_continue_request
+
+# def make_polygon(class_name, point_path):
+#     return Annotation(AnnotationClass(class_name, "polygon"), {"path": point_path})
+
+
+def build_main_annotations_lookup_table(annotation_classes):
+    lookup = {}
+    for cls in annotation_classes:
+        for annotation_type in cls["annotation_types"]:
+            if annotation_type["granularity"] == "main":
+                if annotation_type["name"] not in lookup:
+                    lookup[annotation_type["name"]] = {}
+
+                lookup[annotation_type["name"]][cls["name"]] = cls["id"]
```

Comment:
Would this work? Not sure by heart
```suggestion
    for cls in annotation_classes:
        for annotation_type in cls["annotation_types"]:
            if annotation_type["granularity"] == "main":
                lookup[annotation_type["name"]] = {cls["name"]: cls["id"]}
```

Reply:
It would override other classes for the same type